readthedocs/rtd_tests/tests/test_doc_builder.py

```diff
@@ -362,7 +362,8 @@ def test_get_theme_name_with_feature_flag(self, checkout_path, run):
                     'google_analytics': mock.ANY,
                     'theme': 'readthedocs',
                 },
-                mock.ANY,
+                stream=mock.ANY,
+                Dumper=mock.ANY,
```

Comment:
Maybe we want to check for our `SafeDumper` here to be sure, right?

Reply:
Indeed, good catch.